src/utils.ts

```diff
@@ -124,3 +125,11 @@ export function fixJsonQuotes(target: string): string {
      }
      return result;
 }
+
+export function save(ent: Entity | null, entName: string, timestamp: BigInt): void {
+  if (ent == null) {
+    return;
+  }
+  (ent as Entity).setBigInt('lastUpdatedAt', timestamp);
+  store.set(entName, ent.getString('id'), ent as Entity);
```

Comment:
does ent.save () will not do the work here ?

Reply:
No, it doesn't exist here